src/database/migrations/20240514075609-add-default-entities.js

```diff
@@ -8,6 +8,13 @@ module.exports = {
 		}
 
 		const entitiesArray = [
+			{
+				entityType: 'title',
+				entities: '',
+				has_entities: false,
+				validation: { regex: `[^A-Za-z0-9 <>_&-]`, required: true },
```

Comment:
@priyanka-TL The max allow length should be checked via regex here. For all entity types length has to be checked via regex including NOTES

Reply:
Note limit need to send in config api hence keeping this in env